### ISSUE TITLE
pull major bodies in the major.json file to predict moons position of each planet

### ISSUE THREAD

**Issue by adhithyasimha** [SpaceForce-1/pathfinder#15]

**Comment by adhithyasimha** [comment]
@carlosmonte You would just need to look up the ephemeris data for those bodies. Then you would just need to pick a central location like the center of the planet for the moons and then type in the right mass parameters.

**Comment by cmontalvo251** [comment]
That's right. The code right now assumes that the sun is the center and all the gravitational parameters are based on the location of the planets in reference to the sun and the mass of the sun.

If we wanted to do moons around Jupiter we would just need to change the mass to Jupiter and then look up the orbital elements of all the moons.
